<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWithDateAxisWidget</class>
 <widget class="QMainWindow" name="GraphWithDateAxisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphWithDateAxisWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Positions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="p_dock_Positions">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="p_list_PositionsOnChart">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_CheckAll">
       <property name="text">
        <string>Check all position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_UncheckAll">
       <property name="text">
        <string>Uncheck all position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_updateGraph">
       <property name="text">
        <string>Update chart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
